commission/views/commission_settlement_views.xml

```diff
@@ -0,0 +1,178 @@
+                    name="thisyear"
+                    domain="[('date_from', '&gt;=', time.strftime('%%Y-01-01')), ('date_to', '&lt;=', time.strftime('%%Y-12-31'))]"
+                />
+                <separator />
+                <group expand="1" string="Group By">
+                    <filter
+                        string="Agent"
+                        name="group_agent"
+                        context="{'group_by': 'agent_id'}"
+                    />
+                    <filter
+                        string="Date from month"
+                        name="group_date"
+                        context="{'group_by': 'date_from:month'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="view_settlement_form">
+        <field name="name">Settlements</field>
+        <field name="model">commission.settlement</field>
+        <field name="arch" type="xml">
+            <form string="Settlement" create="false">
+                <header>
+                    <button
+                        string="Cancel"
+                        states="settled"
+                        type="object"
+                        name="action_cancel"
+                    />
+                    <field name="state" widget="statusbar" />
+                </header>
+                <sheet>
+                    <field name="can_edit" invisible="1" />
+                    <group attrs="{'readonly': [('can_edit', '=', False)]}">
+                        <group>
+                            <field name="agent_id" />
+                            <field name="date_from" />
+                            <field name="settlement_type" />
+                        </group>
+                        <group>
+                            <field
+                                name="company_id"
+                                groups="base.group_multi_company"
+                            />
+                            <field name="date_to" />
+                            <field name="currency_id" invisible="1" />
+                            <field name="agent_type" invisible="1" />
+                        </group>
+                    </group>
+                    <group string="Commission lines">
+                        <field
+                            name="line_ids"
+                            nolabel="1"
+                            attrs="{'readonly': [('can_edit', '=', False)]}"
+                            colspan="2"
+                        >
+                            <tree editable="top">
+                                <field name="date" />
+                                <field name="commission_id" />
+                                <field name="settled_amount" />
+                            </tree>
+                        </field>
+                    </group>
+                    <group class="oe_invoice_lines_tab">
```

Comment:
```suggestion
                    <group>
```
I think we should remove this style as it comes from `account` module and doesn't seem to do anything here.

Reply:
Updated